src/main/java/uk/gov/pay/adminusers/resources/UserResource.java

```diff
@@ -90,6 +90,15 @@ public Response getUser(@PathParam("userExternalId") String externalId) {
                 .orElseGet(() -> Response.status(NOT_FOUND).build());
     }
     
+    @GET
+    @Path("/admin-emails-for-gateway-accounts")
+    @Produces(APPLICATION_JSON)
+    @Consumes(APPLICATION_JSON)
+    public Map<String, List<String>> getAdminUserEmailsForGatewayAccountIds(
+            @QueryParam("gatewayAccountId") List<String> gatewayAccountIds) {
```

Comment:
Question on this param being a query string, I believe the default serialisation for this is `gatewayAccountId=xxx&gatewayAccountId-xxx`. 

If the number of unique searchable ids could be ~500, would we run into issues with maximum query string policies enforced anywhere along the app/ load balancer/ app proxies? Potentially using a `POST` body would get around this providing almost any data size with multiple body pages.

Reply:
These are good reasons to switch to using POST with a body. Fixed.